netbox_secrets/api/views.py

```diff
@@ -153,24 +150,7 @@ class GetSessionKeyViewSet(ViewSet):
     permission_classes = [IsAuthenticated]
 
     parser_classes = [JSONParser, FormParser, MultiPartParser]
-    
-    @swagger_auto_schema(
```

Comment:
Is this taken care of automatically? If not, we need this back for OpenAPI spec to recognize the correct schema

Reply:
I test it, and it works fine automatically with the OpenAPI 3.0